### DISCUSSION TITLE
Setting player username

### DISCUSSION THREAD

**Discussion by TheNuclearNexus** [headlesshq/headlessmc#163] [Q&A]
Is it currently possible to the player's username when launching in offline mode?

I didn't seen anything directly in the code base that allows for this, so unless I am missing something a CLI arg I assume no?

**Comment by 3arthqu4ke** [comment]
I might have time to make this feature next week.
https://github.com/3arthqu4ke/headlessmc/blob/61bd90233759209370d3eaf097412f5feaf0b080/headlessmc-launcher/src/main/java/me/earth/headlessmc/launcher/auth/AccountManager.java#L47

**Comment by 3arthqu4ke** [comment]
See #164
